can_bus/json/BMS/BMS_tx.json

```diff
@@ -292,5 +292,48 @@
                 "unit": "W"
             }
         }
+    },
+    "BMS_AuxThermistors": {
+        "msg_id": 121,
+        "cycle_time": 100,
+        "description": "BMS Auxiliary Thermistor Temperatures",
+        "signals": {
+            "ThermTemp0": {
```

Comment:
Are these populated in permanent locations? We should give them better names if so. Fine to leave that for a follow-up though.

Reply:
At the time of testing they had not been, will create a follow up with better labels after consulting with Mallory